src/repl/scala/tools/nsc/interpreter/Power.scala

```diff
@@ -105,17 +106,12 @@ class Power[ReplValsImpl <: ReplVals : ru.TypeTag: ClassTag](val intp: IMain, re
       if (packageClass.isPackageClass)
         apply(packageClass)
       else {
-        repldbg("Not a package class! " + packageClass)
+//        repldbg("Not a package class! " + packageClass)
         Set()
       }
     }
   }
 
-  private def customBanner = replProps.powerBanner.option flatMap {
-    case f if f.getName == "classic" => Some(classic)
-    case f => io.File(f).safeSlurp()
-  }
```

Comment:
That'll be an enterprise feature. Just kidding. It moved a bit closer to the co-source of the banner printer.